UKHomeOffice/nmsw-ui
# NMSW-429 Set focus on summary on error

# Ticket

NMSW-429

----

## AC

Tab order isn't updating correctly.
Set focus on error summary when there are errors on the page

----

## To test

- Run app
- Go to localhost:3000
- Sign in
- Click report a voyage
- On the general declaration page click Check for errors
- > The page should scroll to the top and set focus on the error summary
- Go to Your details
- Click change your details 
- Click continue
- > The page should scroll to the top and set focus on the error summary

----

## Notes

Testing in 2 pages (forms and file upload) as they are handled separately
Also added conditional error class to autocomplete component as the input was missing a red border on error
